### DISCUSSION TITLE
【绝区零live2d】能吃能睡的鲨鱼希人少女！模型大公开，人手一只免费送~

### DISCUSSION THREAD

**Discussion by yfk2012** [share121/inter-knot#1413] [其他]
![QQ截图20240811171232](https://github.com/user-attachments/assets/c35cca08-ae0c-4527-b373-3baa0ac5133e)

原画：[@神宫凉子](https://space.bilibili.com/13737731)
建模：[@杨小唸](https://space.bilibili.com/132134)
视频：[@夜雀Rodeisy](https://space.bilibili.com/267145)
打码暗黑骑士：[@蒜蓉小龙虾Lesiry](https://space.bilibili.com/122464337)
*领取方式在评论区置顶*
【艾莲·乔模型注意事项】
1.艾莲·乔模型的版权归属于miHoYo
2.本模型是画师神宫凉子与建模师杨小唸共同制作，为爱发电，全平台免费，约稿米画师搜：萨小启
3.本模型可以用于制作绝区零相关的视频，使用时请符合官方二创规定
4.本模型仅供娱乐，禁止商用，禁止倒卖，禁止二改
5.本模型可用于直播，但禁止盈利
6.不得用于政治宣传 请符合官方规定使用
7.创作者不为违反规定使用的人负责

【简单使用流程】
1、通过链接下载压缩包
2、根据自己使用的软件（vts、prpr、直播姬），选择相应模型文件，不同模型文件都经过简单配置，打开即用
3、把模型导入面捕软件，开启面捕即可使用
4、直播推流与面捕软件的具体使用方法，可在b站自行搜索

粉丝数过5w从评论区选1个点赞最高的角色立刻做成定制免费模型！

**Comment by gaojiaobei** [comment]
豪！给你点赞

**Comment by yfk2012** [comment]
这是我画师妈妈制作的模型! 请大家多多去支持!
